external-crates/move/crates/move-compiler/src/parser/ast.rs

```diff
@@ -690,6 +690,7 @@ impl Var {
 
     pub fn is_valid_name(s: Symbol) -> bool {
         s.starts_with('_')
+            || (s.starts_with('`') && s.ends_with('`'))
```

Comment:
Who uses this function?

Reply:
Everyone who checks if a local name is valid uses it. Though it is unnecessary, since we do notch them out. Will fix!